.gitignore

```diff
@@ -22,4 +24,14 @@ VERSION
 endpoints.json
 generate-doc
 # Ignore vi backup files
-*~
\ No newline at end of file
+*~
+# Ignore patches.
+*.patch
+# Eclipse files
+.classpath
+.project
+.settings/**
+# Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
+.idea/
+*.iml
+
```

Comment:
I use vim. It would be nice if you could add `*.swp` in place of this empty line :)

Reply:
done